pycbc/workflow/coincidence.py

```diff
@@ -262,7 +262,7 @@ def create_node(self, statmap_files, tags=None):
         node = Node(self)
         node.add_input_list_opt('--statmap-files', statmap_files)
         node.new_output_file_opt(statmap_files[0].segment, '.hdf',
-                                 '--output-file', tags=tags)
+                                 '--output-file', tags=None)
```

Comment:
can you explain why this change to tags=None is needed / desirable

Reply:
To be honest I don't think this is needed and will revert in this case

I did this because changing from `tags=tags` to `None` within the `PyCBCMultiifoExcludeZerolag.create_node()` meant that the tags didn't appear twice in the output filename, and thought this could be an issue elsewhere. It doesn't seem to actually make a difference in this case

Reply:
If the tags are appearing twice in the output filename then it is a problem. I don't think the issue is in this line though. Indeed removing this will just cause the `tags` kwarg to do nothing. I think the issue is that if you have already sent the tags when creating the Executable you should not send the same tags again when creating the Node.